R/input.R

```diff
@@ -4,7 +4,7 @@
 #' @export
 #' @examples
 #' read_file(file.path(R.home(), "COPYING"))
-read_file <- function(file) {
-  ds <- datasource(file)
+read_file <- function(file, skip = 0) {
```

Comment:
Exposing `skip` here doesn't make a lot of sense to me, since `read_file()` is not line based